content/en/platform/corda/5.0-beta/developing/getting-started/utxo-ledger-example-cordapp/running-the-chat-cordapp.md

```diff
@@ -66,55 +66,57 @@ Swagger also gives the curl command which you can use to run the request directl
 
   ```java
 curl -X 'POST' \
-  'https://localhost:8888/api/v1/flow/17F49B05B2B5' \
+  'https://localhost:8888/api/v1/flow/253501665E9D' \
   -H 'accept: application/json' \
+  -H 'Authorization: Basic YWRtaW46YWRtaW4=' \
   -H 'Content-Type: application/json' \
   -d '{
-    "clientRequestId": "create-1",
-    "flowClassName": "com.r3.developers.csdetemplate.utxoexample.workflows.CreateNewChatFlow",
-    "requestData": {
-        "chatName":"Chat with Bob",
-        "otherMember":"CN=Bob, OU=Test Dept, O=R3, L=London, C=GB",
-        "message": "Hello Bob"
-        }
-}'
+  "clientRequestId": "create-1",
+  "flowClassName": "com.r3.developers.csdetemplate.utxoexample.workflows.CreateNewChatFlow",
+  "requestBody": {
+  "chatName":"Chat with Bob",
+  "otherMember":"CN=Bob, OU=Test Dept, O=R3, L=London, C=GB",
+  "message": "Hello Bob"
+  }
+  }'
 ```
```

Comment:
Redundant leading spaces

Reply:
That's the format it comes out of swagger in, (with no indentation) so I chose not to reform.

Reply:
I've corrected it and pushed, but for some reason its not coming through on github verison. Either way, I don't think the spaces render anyway. Will need to check on the actual site preview.